### PR TITLE
Change minimum required RAM from 2GB to 1GB

### PR DESCRIPTION
**Pull request by pcoder**

### CODE REVIEW

File: datacenterlight/static/datacenterlight/js/main.js [hunk 1]
```diff
@@ -15,8 +15,8 @@
         },
         'ram': {
             'id': 'ramValue',
-            'value': 2,
-            'min': 2,
+            'value': 1,
```
Comment (telmich) [inline datacenterlight/static/datacenterlight/js/main.js]: I'd leave the default (value) at 2

Comment (pcoder) [reply datacenterlight/static/datacenterlight/js/main.js]: Thanks for that info. Corrected it now.

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -128,8 +128,8 @@ msgstr "MwSt. inklusive"
 msgid "Please enter a value in range 1 - 48."
 msgstr "Bitte gib einen Wert von 1 bis 48 ein."
 
-msgid "Please enter a value in range 2 - 200."
-msgstr "Bitte gib einen Wert von 2 bis 200 ein."
+msgid "Please enter a value in range 1 - 200."
+msgstr "Bitte gib einen Wert von 1 bis 200 ein."
 
 msgid "Please enter a value in range 10 - 2000."
 msgstr "Bitte gib einen Wert von 10 bis 200 ein."
```
Comment (tiwariav) [inline hosting/locale/de/LC_MESSAGES/django.po]: I think we can correct `200` to `2000` here too.

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -520,6 +520,9 @@ msgstr ""
 "Deine VM ist gleich bereit. Wir senden Dir eine Bestätigungsemail, sobald Du "
 "auf sie zugreifen kannst."
 
+#~ msgid "Please enter a value in range 2 - 200."
+#~ msgstr "Bitte gib einen Wert von 2 bis 200 ein."
+
```
Comment (tiwariav) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: this can be deleted.

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -722,6 +722,9 @@ msgstr ""
 "Es gab einen Fehler bei der Bearbeitung Deine Anfrage. Bitte versuche es "
 "noch einmal."
 
+#~ msgid "Please enter a value in range 2 - 200."
+#~ msgstr "Bitte gib einen Wert von 2 bis 200 ein."
+
```
Comment (tiwariav) [inline hosting/locale/de/LC_MESSAGES/django.po]: this can be deleted too.

Comment (pcoder) [reply hosting/locale/de/LC_MESSAGES/django.po]: Leaving this for a later job. To do a cleanup.

**Pull request merged**